src/index.tsx

```diff
@@ -25,8 +25,8 @@ class P5Wrapper extends React.Component<IP5WrapperProps, IP5WrapperState> {
       ...props,
       canvas: null,
       // TODO: find a workaround since refs in state are bad practice
-      wrapper: this.wrapper
-    }
+      wrapper: this.wrapper,
```

Comment:
Can you update the config to prefer no trailing commas please?

Reply:
Done